llvm/test/tools/llvm-ctxprof-util/Inputs/valid.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "Guid": 1000,
+    "Counters": [1, 2, 3],
+    "Callsites": [
+      [],
+      [
+        {
+          "Guid": 2000,
+          "Counters": [4, 5]
+        },
+        {
+          "Guid": 18446744073709551613,
+          "Counters": [6, 7, 8]
+        }
+    ], 
+    [
+      {
+        "Guid": 3000,
+        "Counters": [40, 50]
+      }
+    ]
+  ]
+  },
+  {
+    "Guid": 18446744073709551612,
+    "Counters": [5, 9, 10]
```

Comment:
I don't see these counter values in the tested output

Reply:
fixed.